tutorials/2_Gazebo_and_ROS.ipynb

```diff
@@ -0,0 +1,1434 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h1><strong><u>Gazebo and ROS</strong></u></h1>\n",
+    "\n",
+    "<h2>What is Gazebo</h2>\n",
+    "\n",
+    "<p>Gazebo is a 3D simulator with the ability to accurately and efficiently simulate populations of robots in complex indoor and outdoor environments. While similar to game engines, Gazebo offers physics simulation at a much higher degree of fidelity, a suite of sensors, and provides both a graphical and a command line interface. In the world of robotics, it is a very popular simulator and has even been selected as the official simulator for the <a href=\"http://www.darpa.mil/program/darpa-robotics-challenge\">DARPA robotics challenge</a>.\n",
+    "</p>\n",
+    "\n",
+    "<p>Typical uses of Gazebo include:</p>\n",
+    "\n",
+    "<ul>\n",
+    "    <li>testing robotics algorithms</li>\n",
+    "    <li>designing robots</li>\n",
+    "    <li>performing regression testing with realistic scenarios</li>\n",
+    "</ul>\n",
+    "    \n",
+    "<p>A few key features of Gazebo include:</p>\n",
+    "\n",
+    "<ul>\n",
+    "    <li>multiple physics engines such as ODE(Open Dynamics Engine), Bullet, Simbody and Dart</li>\n",
+    "    <li>a rich library of robot models and environments, with the option to define your own</li>\n",
+    "    <li>a wide variety of possible sensors</li>\n",
+    "    <li>convenient programmatic and graphical interfaces</li>\n",
+    "</ul>\n",
+    "\n",
+    "<img src=\"http://www.pirobot.org/wordpress/wp-content/uploads/2014/09/gazebo_kinect_challenge.png\" alt=\"\">\n",
+    "<h3 style=\"padding-left:32%\"><u>Mobile robot in a Gazebo environment</u></h3>\n",
+    "\n",
+    "<h1>User Interface</h1>\n",
```

Comment:
The full section from "User Interface" till (not including) "Linking Gazebo" is not showing up!